Source/WebCore/svg/SVGGeometryElement.cpp

```diff
@@ -47,7 +47,7 @@ SVGGeometryElement::SVGGeometryElement(const QualifiedName& tagName, Document& d
 
 float SVGGeometryElement::getTotalLength() const
 {
-    document().updateLayoutIgnorePendingStylesheets();
+    document().updateLayoutForNode(*this);
```

Comment:
Does content-visibility properly work on SVG in general?

Reply:
I have to ask around, since officially svg does not generate boxes, maybe not.

Reply:
Playing around with this in chromium, it seems supported. I'll try to check whether this is by design, but I do think there is no proper test for this, since all tests seem to use outer svg or its container for setting content-visibility, not the svg shapes/text/groups etc.